client/blocks/authentication/social/style.scss

```diff
@@ -0,0 +1,100 @@
+@import "@automattic/onboarding/styles/mixins";
+@import "@wordpress/base-styles/breakpoints";
+@import "@wordpress/base-styles/mixins";
+
+.auth-form__social {
+	box-shadow: none;
+
+	.auth-form__social-buttons {
+		display: flex;
+		flex-direction: column;
+		height: 100%;
+		justify-content: center;
+	}
+
+	.social-buttons__button {
+		text-align: start;
+		box-shadow: none;
+		background-color: transparent;
+
+		@include break-small {
+			text-align: center;
+		}
+
+		@include break-medium {
+			text-align: start;
+		}
+	}
+}
+
+.auth-form__social.is-signup {
+	.social-buttons__button {
+		padding-left: 0;
+		padding-bottom: 0;
+	}
+}
+
+.auth-form__social.is-login {
+	p:first-of-type {
+		display: none;
+	}
```

Comment:
Can we give this `p` a classname? It will be a bit mysterious if someone adds a paragraph to the form but it won't render.